test/e2e/app-dir/app-static/app/api/draft-mode/route.ts

```diff
@@ -1,6 +1,8 @@
 import { draftMode } from 'next/headers'
 import { NextRequest, NextResponse } from 'next/server'
 
+export const revalidate = 1
```

Comment:
I think technically this should be dynamic too since it uses `searchParams` from `req`